src/main/java/org/springframework/data/couchbase/core/ReactiveRemoveByQueryOperationSupport.java

```diff
@@ -75,7 +68,7 @@ public ReactiveRemoveByQueryOperationSupport(final ReactiveCouchbaseTemplate tem
 								return throwable;
 							}
 						}).flatMapMany(ReactiveQueryResult::rowsAsObject)
-						.map(row -> new RemoveResult(row.getString("id"), row.getLong("cas"), Optional.empty()));
+						.map(row -> new RemoveResult(row.getString("__id"), row.getLong("__cas"), Optional.empty()));
```

Comment:
The names (__id, __cas) should come from TemplateUtils, although they seem to be hard-coded in several other places.

Reply:
Fixed